docs/orchestrating-elastic-stack-applications/beat.asciidoc

```diff
@@ -0,0 +1,403 @@
+:page_id: beat
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Beats on ECK
+
+This section describes how to configure and deploy Beats with ECK.
+
+* <<{p}-beat-quickstart,Quickstart>>
+* <<{p}-beat-configuration,Configuration>>
+* <<{p}-beat-troubleshooting,Troubleshooting>>
+
+[id="{p}-beat-quickstart"]
+== Quickstart
+
+. Apply the following specification to deploy Filebeat. ECK automatically configures the secured connection to an Elasticsearch cluster named `quickstart`, created in the link:k8s-quickstart.html[Elasticsearch quickstart].
++
```

Comment:
I was thinking our quickstart would be more like this (less yaml):

```
Run the following specification to deploy Filebeat and Metricbeat on your Kubernetes cluster. Both are setup to collect logs from all Pod, metrics from all hosts, and ship data to an Elasticsearch cluster named `quickstart`, created in the link:k8s-quickstart.html[Elasticsearch quickstart].

kubectl apply -f github.com/elastic/cloud-on-k8s/link/to/k8s-observability-recipe.yaml
```

Reply:
It is fairly short for what it does (compared to non-eck approach), so I thought it might be good to show that. If we just have a link, then it's not immediately obvious whether it's just our resource or also bunch of other things. WDYT? We will also have a recipes section that will have links only.

Reply:
I think I have changed my mind and agree with David that it is actually short enough to be included directly.

Reply:
👍 fine with me as well, it has the benefit of making people understand the CRD logic.
Let's make sure we include a link to the recipes (once we have them) in the quickstart so people can jump quickly to the interesting manifests.